src/main/java/com/example/demo/client/PostObject.java

```diff
@@ -0,0 +1,14 @@
+package com.example.demo.client;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class PostObject {
+    int USER_ID;
```

Comment:
1) set all the fields as `private`
2) follow java naming conventions
3) use wrapper types instead - default values could confuse us
4) you can use just `Post` as the name of the class

Reply:
https://www.oracle.com/technetwork/java/codeconventions-150003.pdf